src/lib/components/Footer/Footer.svelte

```diff
@@ -0,0 +1,164 @@
+<script lang="ts">
+  import "./Footer.scss";
+
+  import { url as facebookIcon } from "$icons/facebook";
+  import { url as youtubeIcon } from "$icons/youtube";
+
+  import Icon from "$lib/components/Icon";
+  import Link from "$lib/components/Link";
+  import Logo from "$lib/components/Header/Logo";
+
+  import type { AfterNavigate } from "@sveltejs/kit";
+  import type { NavMenuType } from "$lib/components/Header/Nav";
+  import type { SiteTitleType } from "$lib/components/Header/Title";
+
+  export let navItems: NavMenuType[] = [];
+  export let siteTitle: SiteTitleType;
+  // we provide this from a parent component so that we can render the component in Storybook
+  export let afterNavigate: undefined | ((callback: (navigation: AfterNavigate) => void) => void) =
+    undefined;
+
+  // Quickest way to implement the accordions was to replicate what USWDS was
+  //   doing, which is to use buttons on mobile and headings on larger screens.
+  const MOBILE_BREAKPOINT = 480;
+  $: innerWidth = 0;
+
+  // TODO: Logic for accordions here is very similar to that in
+  //       Header/Nav/NavMenu; we should make it reusable.
+  let expandedIndex: number | undefined = undefined;
+
+  const toggle = (i: number) => {
+    expandedIndex = expandedIndex === i ? undefined : i;
+  };
+
+  const handleKeyDown = (event: KeyboardEvent, i: number) =>
+    (event.key === "Enter" || event.key === " ") && toggle(i);
+  const handleMouseDown = (i: number) => toggle(i);
+
+  if (afterNavigate) afterNavigate(() => (expandedIndex = undefined));
```

Comment:
I would have preferred to use `navigating` (from `$app/stores`) but that gets set to true as soon as the user begins navigation, which seemed a bit disorienting (user would click, things would shift due to collapsing menus, then page would load).